internal/handler/user_service.go

```diff
@@ -0,0 +1,101 @@
+package handler
+
+import (
+	"context"
+
+	"github.com/gobwas/glob"
+	"google.golang.org/grpc/codes"
+	"google.golang.org/grpc/status"
+
+	"github.com/smailic05/TaskInfoblox/internal/pb"
+)
+
+type User struct {
+	Address  string
+	Username string
+	Phone    string
+}
+
+type UserService struct {
+	pb.UnimplementedUserServiceServer
+	UserSlice []User
+}
+
+func New() *UserService {
+	return &UserService{UserSlice: make([]User, 0)}
+}
+
+func (s *UserService) AddUser(ctx context.Context, addUser *pb.AddUserRequest) (*pb.AddUserResponse, error) {
```

Comment:
`error` is always nil here. Do we really need 2 return values here?

Reply:
due to changes, the function now returns error